src/dynamic_types.h

```diff
@@ -183,7 +183,7 @@ typedef struct private_subformat_data
 # endif
 # endif
 # define LOOP_STR
-# if SIMD_COEF_32 == 4
+# if SIMD_COEF_32 >= 4
 #  ifdef MD5_SSE_PARA
 #   define ALGORITHM_NAME		"128/128 " MD5_SSE_type  " " STRINGIZE(BY_X) "x4x" STRINGIZE(MD5_SSE_PARA)
 #   define BSD_BLKS (MD5_SSE_PARA)
```

Comment:
All the above should probably have their own clauses for AVX2 and for MIC, where the latter might need lower OMP_SCALE.